### ISSUE TITLE
Fechar conexão com o banco de dados nas implementações dos DAO

### ISSUE THREAD

**Issue by andersonfernandes** [CC20152/sisufal#14]
Eu estava dando uma olhada nas implementações dos DAO e percebi que as conexões feitas com o banco não estão sendo fechadas depois das consultas. Isso pode gerar um problema grave quando a aplicação for pra produção.
O ideal é que a conexão seja fechada logo após a execução da consulta, ou em um bloco finally. 
A classe Conexao já tem um método desconectar com essa finalidade.

**Comment by dayvsonsales** [comment]
sim, tenta implementar um hook para isto usando algo parecido com aspectos ou vamos ter que mudar todos os dao

**Comment by andersonfernandes** [comment]
Acho que seria mais seguro que sejam feitas as mudanças nos DAO.
